src/kernels/kernelStartupCodeProvider.node.ts

```diff
@@ -32,12 +32,7 @@ export class KernelStartupCodeProvider implements IStartupCodeProvider {
         ) {
             return [];
         }
-        if (
-            !(
-                isLocalConnection(kernel.kernelConnectionMetadata) &&
-                !this.configService.getSettings(undefined).forceIPyKernelDebugger
```

Comment:
Looks like a condition was accidentally added while refactoring the code.
This belongs to startup code of debuggers, not regular code.